apps/backend/src/build-notification/notification.ts

```diff
@@ -18,8 +20,48 @@ export const NotificationPayloadSchema = z.object({
 });
 export type NotificationPayload = z.infer<typeof NotificationPayloadSchema>;
 
-function getStatusContext(buildName: string): string {
-  return buildName === "default" ? "argos" : `argos/${buildName}`;
+/**
+ * Check if the project has a sibling project with the same repository.
+ */
+async function checkHasSiblingProject(project: Project): Promise<boolean> {
+  if (!project.githubRepositoryId && !project.gitlabProjectId) {
+    return false;
+  }
+
+  const query = Project.query();
+
+  if (project.githubRepositoryId) {
+    query.orWhere("githubRepositoryId", project.githubRepositoryId);
+  }
+
+  if (project.gitlabProjectId) {
+    query.orWhere("gitlabProjectId", project.gitlabProjectId);
+  }
```

Comment:
First one should be "where". Second one should be orWhere.
Maybe it's ok to use "orWhere" for the first one too.

Reply:
Yeah it's ok, it's better like that because it's aligned.